src/Resources/views/tailwind_2/_header.html.twig

```diff
@@ -28,27 +28,26 @@
                         data-transition-leave-to="opacity-0 scale-95"
                         tabindex="-1"
                     >
-                        <div class="space-y-2" {% if table.exporters|length > 1 %} {{ stimulus_controller('araise/table-bundle/exporter') }} {% endif %}>
+                        <div {% if table.exporters|length > 1 %} {{ stimulus_controller('araise/table-bundle/exporter') }} {% endif %}>
```

Comment:
Putting If-Elses inline into a HTML Tag is still a bad idea for any sorts of formatting. We need to use other approaches. This is not maintainable and sustainable.

Instead, we could but the stimulus controller on it anyway, and disable it with a stimulus controller value attribute. Much cleaner to read, with no drawbacks.

Reply:
@renestalder I'm against loading Stimulus Controller which are not needed. Those should not be initiated at all if unused.

The same as we should use `html_classes`, we should also do something similar with attributes and following Stimulus Controllers. Although I'm not sure if something like that exists.

Reply:
@marcwieland95 Well, me too, but I guess the current setup is anyway creating one big chunk which is already loaded by the browser, we would have saved nothing from the conditional applying of the controller.

However, with the refactoring now, the feedback is obsolete.